src/modules/integrator-guide/pages/corteza-studio/index.adoc

```diff
@@ -0,0 +1,139 @@
+include::ROOT:partial$variables.adoc[]
+
+= {PRODUCT_NAME} Studio
+
+:description: {PRODUCT_NAME} Studio allows users to customize {PRODUCT_NAME} web applications styling according to their preferences.
+{PRODUCT_NAME} Studio allows users to customize {PRODUCT_NAME} web applications styling according to their preferences. 
+
```

Comment:
I'd personally like you to paste the `:description:` contents into the body (as the document intro)